src/components/Navigator/NavigatorCard.vue

```diff
@@ -0,0 +1,626 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2021 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div class="navigator-card">
+    <div class="head-wrapper">
+      <button class="close-card-mobile" @click="$emit('close')">
+        <InlineCloseIcon class="icon-inline close-icon" />
+      </button>
+      <Reference :url="technologyPath" class="navigator-head">
+        <NavigatorLeafIcon :type="type" with-colors class="card-icon" />
+        <div class="card-link">
+          {{ technology }}
+        </div>
+      </Reference>
+    </div>
+    <div class="card-body">
+      <RecycleScroller
+        v-show="nodesToRender.length"
+        ref="scroller"
+        class="scroller"
+        :items="nodesToRender"
+        :item-size="itemSize"
+        key-field="uid"
+        v-slot="{ item }"
+      >
+        <NavigatorCardItem
+          :item="item"
+          :filter-pattern="filterPattern"
+          :is-active="item.uid === activeUID"
+          :is-bold="activePathMap[item.uid]"
+          :expanded="openNodes[item.uid]"
+          @toggle="toggle"
+          @toggle-full="toggleFullTree"
+        />
+      </RecycleScroller>
+      <div class="no-items-wrapper" v-if="!nodesToRender.length">
+        <template v-if="filterPattern">
+          No results matching your filter
+        </template>
+        <template v-else>
+          Technology has no children
+        </template>
+      </div>
+    </div>
+    <div class="filter-wrapper">
+      <div class="navigator-filter">
+        <div class="input-wrapper">
+          <FilterIcon class="icon-inline filter-icon" :class="{ colored: filter }" />
```

Comment:
This is kind of a subjective comment, but this icon looks clickable to me for some reason even though I know it isn't.

Reply:
The filter would be replaced by - https://github.com/apple/swift-docc-render/pull/49, so lets not focus on that right now.